<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #ADD8E6; /* 浅蓝色背景 */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>上升</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>下降</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lightButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>照明开/关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uvButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>紫外线开/关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>风干开/关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dryButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>烘干开/关</string>
    </property>
   </widget>
   <widget class="QLabel" name="uvLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="windLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="dryLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windButtonTimer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>风干定时3s/次</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dryButtonTimer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #F4A460; /* 沙棕色背景 */
    color: white;
    border: 2px solid #000;
    border-radius: 5px;
    padding: 5px;
}

QLabel {
    color: red; /* 红色字体 */
    font-size: 20px;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>烘干定时3s/次</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
